src/main/webapp/map3.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Marker Storage</title>
+    <link rel = "stylesheet" href = "/css/main.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBZkWQmagGmCHKRX2osG0BSWiUYLOZaxWA"></script>
+    <script>
+        function createMap(){
+            map = new google.maps.Map(document.getElementById('map'), {
+                center: {lat: 38.5949, lng: -94.8923},
+                zoom: 4
+            });
+             // When the user clicks in the map, show a marker with a text box the user can edit.
+            map.addListener('click', (event) => {
+                createMarkerForEdit(event.latLng.lat(), event.latLng.lng());
+            });
+            fetchMarkers();
+        }
+        
+
+
+
+    </script>
+    <style>
+        #map {
```

Comment:
you can create a file called map3.css and just copy paste the line starting with '#map' into it and then include in the header (after "<link rel = "stylesheet" href = "/css/main.css">") <link rel = "stylesheet" href = "/css/map3.css"> to put this in the css file and link the html file to it :)

Reply:
Ok! I will do that